website/docs/overview/query/advanced-query.md

```diff
@@ -1,3 +1,6 @@
+import Tabs from '@theme/Tabs';
```

Comment:
Be sure to change all files where the components are added from .md to .mdx
This file is throwing markdown errors on my machine

Reply:
done.